rust/tvm/examples/resnet/build.rs

```diff
@@ -23,6 +23,7 @@ use std::{path::Path, process::Command};
 fn main() -> Result<()> {
     let output = Command::new("python3")
         .arg(concat!(env!("CARGO_MANIFEST_DIR"), "/src/build_resnet.py"))
+        //.arg(&format!("--pretrained"))
```

Comment:
Left in by accident?

Reply:
No, I intent to add this. Someone who wants to use a pre-trained model could just uncomment this line. I think this is more convenient.

Reply:
Thanks @intheworld, I also agree that we should keep the pretrained flag as it is

Reply:
Ok, I will remove this line.